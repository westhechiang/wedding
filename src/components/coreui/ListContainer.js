import styled from 'styled-components';
import { theme } from '../../ui';

const ListContainer = styled.ul`
  width: ${props => props.width || 'auto'};
  background-color: ${props => theme.color[props.backgroundColor] || 'transparent'};
  max-height: 450px;
  overflow-y: auto;
  height: 100%;
  margin: 0;
  padding: 0;
`;

export default ListContainer;
