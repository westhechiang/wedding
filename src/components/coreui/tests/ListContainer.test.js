import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ListContainer from '../../../components/coreui/ListContainer';

describe('<ListContainer />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<ListContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
