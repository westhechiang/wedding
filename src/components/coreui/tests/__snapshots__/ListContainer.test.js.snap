// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListContainer /> should match its empty snapshot 1`] = `
<ul
  className="sc-bdVaJa beoWEL"
/>
`;
